### PR TITLE
[onnx] Unstrip script

### PR DESCRIPTION
**Pull request by disktnk**
In order to run with onnx runtimes even if the target ONNX are stripped.

```
$ python -m pytorch_pfn_extras.onnx.unstrip_tensor target.onnx
$ python -m pytorch_pfn_extras.onnx.unstrip_tensor target_dir/
```

TODO
* [x] support `*.onnx`
* [x] support `*.pb`
* [x] support directory, unstrip `model.onnx`, input/output at once
* [x] unit test
* [x] docs

### CODE REVIEW

File: pytorch_pfn_extras/onnx/unstrip_tensor.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+def _unstrip_graph(graph: onnx.GraphProto) -> None:
+    for init in graph.initializer:
+        if not strip._is_stripped_or_set_external(init):
+            continue
+        _unstrip_tensor(init)
+
+    for node in graph.node:
+        for attr in node.attribute:
+            if attr.HasField("g"):
+                _unstrip_graph(attr.g)
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/unstrip_tensor.py:51]: I've thought of also un-stripping tensor attributes like following:
```suggestion
            if attr.HasField("g"):
                _unstrip_graph(attr.g)
            elif attr.HasField("t"):
                _unstrip_tensor(attr.t)
```
But current strip doesn't do it so never mind
https://github.com/pfnet/pytorch-pfn-extras/blob/785cc23138a0a648a83dc90c4f09302dc9f0938d/pytorch_pfn_extras/onnx/strip_large_tensor.py#L59

File: pytorch_pfn_extras/onnx/unstrip_tensor.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            file path.
+        out_path (str): Output path to be written.
+    """
+    target_path = Path(path)
+    if not target_path.exists():
+        print(f"Error: the target path is not found, {path}")
+        return None
+    written_path = Path(out_path if out_path else path)
+
+    if target_path.is_dir():
```
Comment (take-cheeze) [inline pytorch_pfn_extras/onnx/unstrip_tensor.py:85]: https://github.com/disktnk/pytorch-pfn-extras/pull/1 meta.json should be also updated

Comment (disktnk) [reply pytorch_pfn_extras/onnx/unstrip_tensor.py:85]: supported to handle `meta.json` if existed

File: pytorch_pfn_extras/onnx/unstrip_tensor.py (hunk 1 above)
Comment (disktnk) [reply pytorch_pfn_extras/onnx/unstrip_tensor.py:51]: it's better to support `attr.t`. will try dealing with another PR along with `strip` method.

**Review by take-cheeze — APPROVED** [review]
LGTM 👍

**Review by kmaehashi — APPROVED** [review]

**Pull request merged**